src/components/Facets.tsx

```diff
@@ -0,0 +1,88 @@
+import { Facets as FacetsCompoundComponent } from './Filters/Facets';
+import { FilterGroup } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment } from 'react';
+
+/**
+ * The CSS class interface for {@link Facets}.
+ *
+ * @public
+ */
+export interface FacetsCssClasses {
```

Comment:
can this be an extension of the FilterGroupCssClasses interface (and similarly for StaticFiltersCssClasses)?

Reply:
sure thing